Search.jsp

```diff
@@ -105,13 +105,16 @@ div
 
 footer
 {
-	background:black;
+	background:dodgerblue;

```

Comment:
Dodgerblue looks better than black as a footer background?

Hui

Reply:
Yes, it looks better.